tests/unit/index.test.js

```diff
@@ -31,3 +31,36 @@ describe('All Methods are Callble', function () {
     })
   })
 })
+
+describe('C2B', function () {
+  const productionInstance = new Mpesa({
+    consumerKey: 'test',
+    consumerSecret: 'test',
+    environment: 'production'
+  })
+
+  it('should throw error in production', function () {
+    let threwError = false
+    
+    try {
+      productionInstance.c2bSimulate()
+    } catch (e) {
+      threwError = true
+      expect(e.message).to.be('Cannot call C2B simulate in production.')
+    } finally {
+      expect(threwError).to.be(true)
+    }
+  })
+
+  it('should not throw error', function () {
+    let threwError = false
+    
+    try {
+      instance.c2bSimulate().then(() => {}).catch(() => {})
```

Comment:
`then` and `catch`  prevent `UnhandledPromiseRejectionWarning`

Reply:
The problem is that you need an actual valid working Mpesa with correct credentials. Check out https://github.com/safaricom/mpesa-node-library/blob/master/tests/helpers/instance.js which gives one a valid test instance

Reply:
Ideally you dont need the try..

Reply:
Seems your right. I've also noticed that calling `instance.c2bSimulate()` without parameters will always lead to and `UnhandledPromiseRejectionWarning`. Using valid credentials and passing args resolves the issue.

Reply:
I'll look for an alternative way that doesn't need a try

Reply:
Actually, just removing it works.